src/handleAction.js

```diff
@@ -5,9 +5,13 @@ function isFunction(val) {
 }
 
 export default function handleAction(type, reducers) {
+  const typeValue = isFunction(type)
```

Comment:
Yes, the `String()` function converts the passed value to a string, calling the `toString()` method if necessary.